code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -203,6 +208,7 @@
 /datum/reagent/toxin/mindbreaker
 	name = "Mindbreaker Toxin"
 	id = "mindbreaker"
+	value = 1
```

Comment:
Is a toxin that stops SRD, usefull for the lesser minded out their

Reply:
What you may want to know is that mindbreaker toxin has a positive reaction rate, so 90 units of reagent makes for 150 units of mindbreaker toxin, meaning that a 1 button roundstart macro nets you 150 units of mindbreaker toxin, for an actual value of 1.6 per unit spend in the machine.

Reply:
Im ok with that.